src/core/EntrySearcher.cpp

```diff
@@ -149,6 +149,8 @@ bool EntrySearcher::searchEntryImpl(Entry* entry)
     auto attributes = QStringList(attributes_keys + entry->attributes()->values(attributes_keys));
     auto attachments = QStringList(entry->attachments()->keys());
 
+    QString entrySizeKiB = QString::number(entry->size() / 1024.0, 'f', 2);
```

Comment:
KiB is kind of obscure for most people, recommend just KB. Additionally, the size value should scale with the size, similar to a file system. If the size is over 1000 KB then it's time to show 1 MB. Same with B, prefer to not have 0.08 KB, should be 80 B.

Reply:
Actually, I would prefer KiB simply for the reason that it tends to be the default unit on all operating systems. Only hard disk manufacturers use base 1000.

Reply:
Sizes have been reverted to B, KiB, etc.